traj/package.xml

```diff
@@ -9,4 +9,6 @@
   <license>Apache 2.0</license>
 
   <buildtool_depend>catkin</buildtool_depend>
+
+  <exec_depend>python-sympy</exec_depend>
```

Comment:
```suggestion
  <exec_depend>python-matplotlib</exec_depend>
  <exec_depend>python-numpy</exec_depend>
  <exec_depend>python-sympy</exec_depend>
```

Reply:
I manually made this change to the package.xml